onnxruntime/core/optimizer/conv_add_act_fusion.cc

```diff
@@ -122,7 +124,7 @@
     return is_supported_cpu_ep_activation(activation_node);
   }
 
-  const Node* SelectProducerConv(const Node& node) const {
+  const Node* SelectProducerConv(const Node& node, bool isNhwc) const {
```

Comment:
>isNhwc [](http://example.com/codeflow?start=56&length=6)

is this necessary?

Reply:
Nice catch, I am using runtimeState.selected_nodes.Target().OpType(); for OpType() function from FuseConvAddActivationAction, which has eliminated the need for additional boolean flag as well.